### PR TITLE
Malkavian Dracula Flow

### PR DESCRIPTION
**Pull request by SilvinaFulp**
## About The Pull Request
Adds quotes from Dracula flow to malkav revelations.
## Why It's Good For The Game
It'd be funny to see a bloodsucker be outted because of the zaza.
## Changelog
Adds more quotes to Clan Malkavian.
:cl:
add: Added more things
/:cl:

### CODE REVIEW

**Comment by SilvinaFulp** [comment]
Ignore the 'Add files via upload'

**Review by Momo8289 — APPROVED** [review]

**Comment by SgtHunk** [comment]
why did u add a whole new file that is an exact copy of the revelations file

File: malkavian_revelations.json
Comment (Momo8289) [inline malkavian_revelations.json:1]: what sgnthunk said, why's this duplicate here?

**Review by Momo8289 — CHANGES_REQUESTED** [review]

**Comment by SilvinaFulp** [comment]
Man where's the delete button

**Comment by JohnFulpWillard** [comment]
```
Adds more quotes to Clan Malkavian.
🆑
add: Added more things
/:cl:
```

should be

```
🆑
add: Added more quotes to Clan Malkavian.
/:cl:
```

**Pull request closed without merging**